### PR TITLE
Updating config comments for v1.9.0 docs

### PR DESCRIPTION
**Pull request by mativm02**
## Description

Updating config comments for v1.9.0 docs
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). If pulling from your own
      fork, don't request your `master`!
- [ ] Make sure you are making a pull request against the **`master` branch** (left side). Also, you should start
      *your branch* off *our latest `master`*.
- [ ] My change requires a change to the documentation.
  - [ ] If you've changed APIs, describe what needs to be updated in the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] Modules and vendor dependencies have been updated; run `go mod tidy && go mod vendor`
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] Check your code additions will not fail linting checks:
  - [ ] `go fmt -s`
  - [ ] `go vet`

### CODE REVIEW

**Comment by buger** [comment]
**API tests result - postgres15-sha256 env: success** :white_check_mark: 
Branch used: refs/pull/794/merge
Commit:  
Triggered by: pull_request (@mativm02)
[Execution page](https://github.com/TykTechnologies/tyk-pump/actions/runs/7830162641)

**Comment by buger** [comment]
**API tests result - mongo44-sha256 env: success** :white_check_mark: 
Branch used: refs/pull/794/merge
Commit:  
Triggered by: pull_request (@mativm02)
[Execution page](https://github.com/TykTechnologies/tyk-pump/actions/runs/7830162641)

File: pumps/pump.go [hunk 1]
```diff
@@ -61,6 +61,7 @@ func processPumpEnvVars(pump Pump, log *logrus.Entry, cfg interface{}, defaultEn
 			log.Error(fmt.Sprintf("Failed to process environment variables for %s pump %s with err:%v ", envVar, pump.GetName(), overrideErr))
 		}
 	} else {
+		fmt.Println("usando defaultEnv:", defaultEnv)
```
Comment (sredxny) [inline pumps/pump.go:64]: please remove

**Review by sredxny — CHANGES_REQUESTED** [review]

**Review by dcs3spp — APPROVED** [review]
Hi @mativm02 Matias, I've read through the PR and all looks good

Which source file does [TYK_PMP_UPTIMEPUMPCONFIG_ENVPREFIX](https://github.com/TykTechnologies/tyk-docs/pull/4066/files/58b7af9e7b1be17f3bc07d1b526e0bb29def6191#r1474670322) appear in? Is it config.go in UptimePumpConfig? In the original PR it was missing a [description](https://github.com/TykTechnologies/tyk-docs/pull/4066/files/58b7af9e7b1be17f3bc07d1b526e0bb29def6191#r1474670322)

**Review by sredxny — APPROVED** [review]

**Comment by buger** [comment]
**API tests result - postgres15-murmur64 env: success** :white_check_mark: 
Branch used: refs/heads/master
Commit: 1b450dff4a0cc7b9746e2ebd06cdea38c3a229ef Updating config comments for v1.9.0 docs (#794)

* Updating config comments

* linting

* removing debug comment

* Adding all default values for EnvPrefix in BaseMongoConf
Triggered by: push (@mativm02)
[Execution page](https://github.com/TykTechnologies/tyk-pump/actions/runs/7831721332)

**Comment by buger** [comment]
**API tests result - mongo44-murmur64 env: success** :white_check_mark: 
Branch used: refs/heads/master
Commit: 1b450dff4a0cc7b9746e2ebd06cdea38c3a229ef Updating config comments for v1.9.0 docs (#794)

* Updating config comments

* linting

* removing debug comment

* Adding all default values for EnvPrefix in BaseMongoConf
Triggered by: push (@mativm02)
[Execution page](https://github.com/TykTechnologies/tyk-pump/actions/runs/7831721332)

**Pull request merged**